subworkflows/local/map.nf

```diff
@@ -29,14 +47,6 @@ workflow MAP {
                                     index: [ meta2, index]
```

Comment:
```suggestion
                                    index: [ meta2, index] 
```

Note the clone above again

Reply:
`new_meta = meta + [ reference: meta2.id ]`?
and why do we need a space at the end here?

Reply:
pretty sure he meant to put it before the bracket

Reply:
Makes sense, thanks!

Reply:
I did indeed :D